deploy/sre-build-test/50-build-test.CronJob.yaml

```diff
@@ -79,6 +86,12 @@ spec:
                     ;;
                   Complete)
                     echo "$(date): Build Complete"
+                    JOBS_TO_DELETE=$(oc -n "${POD_NS}" get job --no-headers || true | \
```

Comment:
Should these deletion steps go in the cleanup trap instead?

Reply:
that could happen but my bash-foo is not that string and my filtering do not work inside the trap 😊 , maybe I will be able to solve it later

Reply:
Actually @cblecker, we want this to happed only on success, so putting it in trap is counter intuative

Reply:
resolving according to @NautiluX response on slack